salt/metalk8s/addons/logging/loki/deployed/chart.sls

```diff
@@ -184,7 +184,7 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: f6917e98336282f93b8cf80b99ca1e78f7adfd297b98c367c1cc420a1cb8d1ac
+        checksum/config: f9c7883a31a1ef8ef3264cfb5160f3cc015a91cf46c6687d7f2777a62af6936b
```

Comment:
Not related to this PR but since we have CSC for loki configuration this one should be computed by salt.
And it's the same for alertmanager, prometheus and grafana I think (+ maybe others)

Reply:
Good point, I'll add it in this PR if that's OK for you

Reply:
So, Prometheus doesn't have a "config file" per-se, it's entirely managed through CRs by prometheus-operator, so I don't have something to compute a checksum from.
Did it for Grafana and Alertmanager though (even if the latter is also managed by prometheus-operator), and Loki of course.